packages/taquito-local-forging/src/schema/operation.ts

```diff
@@ -126,6 +126,16 @@ export const TxRollupSubmitBatchSchema = {
   burn_limit: CODEC.BURN_LIMIT,
 };
 
+export const IncreasePaidStorageSchema = {
+  source: CODEC.PKH,
+  fee: CODEC.ZARITH,
+  counter: CODEC.ZARITH,
+  gas_limit: CODEC.ZARITH,
+  storage_limit: CODEC.ZARITH,
+  amount: CODEC.ZARITH,
+  destination: CODEC.ADDRESS,
```

Comment:
The ADDRESS codec can encode implicit and smart contract addresses. Destination, in this case, should be a smart contract. Should we have a dedicated smartContractAddressEncoder?

Reply:
that's a good point, let's add that

Reply:
@roxaneletourneau I've added the new encoder/decoder

Reply:
@roxaneletourneau oops let me add some unit tests for the new encoder